docs/ocis/release_notes.md

```diff
@@ -0,0 +1,282 @@
+    - Copying existing public links
+    - Editing existing public links
+    - Removing existing public links
+  - Viewing public links
+
+{{< /tab >}}
+{{< tab "User Profile" >}}
+#### User Profile
+- Display basic profile information (user name, display name, e-mail, group memberships)
+- "Edit" button guides to ownCloud 10 user settings (when used with oC 10)
+
+{{< /tab >}}
+{{< tab "User Settings" >}}
+#### Basic user settings
+- Language of the web interface
+
+{{< /tab >}}
+{{< /tabs >}}
+
+### oCIS Backend Features
+
+{{< tabs "backend-features" >}}
+{{< tab "Storage" >}}
+#### Storage
+@TODO
+
+{{< /tab >}}
+{{< tab "IDM" >}}
+#### User and group management
+- Functionality available via API and frontend ("Accounts" extension)
+- User listing (API/FE)
+- User creation (API/FE)
+- User deletion (API/FE)
+- User activation/blocking (API/FE)
+- Role assignment for users (API/FE)
+- User editing (API)
+- Multi-select in the frontend (delete & block/activate)
+- Group creation (API)
+- Add/remove users to/from groups (API)
+- Group deletion (API)
+- Create/read/update users and groups (CLI)
+
+{{< /tab >}}
+{{< tab "Settings" >}}
+#### Settings
+##### Settings bundles framework
+- What is a settings bundle?
+- What can you do with it?
+- Extensions?
+
+##### Roles & Permissions System
+Infinite Scale follows a role-based access control model. Based on permissions for actions which are provided by the system and by extensions, roles can be composed. Ultimately, these roles can be assigned to users to define what users are permitted to do. This model allows a segregation of duties for administration and allows granular control of how different types of users (e.g., Guests) can use the platform.
+
+- Currently available permissions: Manage accounts (gives access to the internal user management)
+- The current roles are exemplary default roles which are defined in config files
+  - "Admin": Has the permission to "manage accounts"
+  - "User": Does not have any dedicated permission
+  - "Guest": Does not have any dedicated permission
+- Currently a user can only have one role
+- Users with the role "Admin" can assign/unassign roles to/from other users (as part of the permission to "manage accounts")
+
+{{< /tab >}}
+{{< tab "APIs" >}}
+#### APIs
+- WebDAV
+- OCS
+
+{{< /tab >}}
+{{< /tabs >}}
+
+### Known issues for OCIS standalone
+- There are feature differences depending on the operation mode, e.g., no user management with ownCloud Web and oC 10 backend
+- Public links do not yet respect the given role (a recipient has full permissions no matter which role has been set)
+- Resharing works but does not have necessary restrictions in place
+  - Share recipients can add more people or create public links with higher permissions than they originally had
+  - Every person in a share can see all other people in the people list
+- Sharing indicators in the file list will only be shown after opening the right sidebar for a resource
```

Comment:
This is only true for public links. The `people` share indicators are displayed right away.